src/lib/containers/table/SynapseTable.tsx

```diff
@@ -329,7 +328,7 @@ export default class SynapseTable extends React.Component<
       unitDescription,
       token,
       showBarChart,
-      enableLeftFacetFilter,
+      topLevelControlsState,
```

Comment:
Special prop unnecessary, it has this information already from the QueryWrapperChildProps, so use it instead.